channels-rke2.yaml

```diff
@@ -366,6 +366,30 @@ releases:
       rke2-ingress-nginx:
         repo: rancher-rke2-charts
         version: 4.0.307
+  - version: v1.21.12-rc2+rke2r1
+    minChannelServerVersion: v2.6.3-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v1-21-12-rc2-rke2r1
+      <<: *serverArgs-v1-21-10-rke2r2
+    agentArgs: &agentArgs-v1-21-12-rc2-rke2r1
+      <<: *agentArgs-v1-21-10-rke2r2
+    charts: &charts-v1-21-12-rc2-rke2r1
+      <<: *charts-v1-21-11-rke2r1
```

Comment:
The server and agent args should also pull from v1-21-11-rke2r1 otherwise they will miss some of the changes that were present from those.